src/PlaywrightSharp.Firefox/FirefoxPage.cs

```diff
@@ -321,6 +339,23 @@ private void OnExecutionContextDestroyed(RuntimeExecutionContextDestroyedFirefox
             }
         }
 
+        private void RemoveContextsForFrame(object sender, FrameEventArgs e)

```

Comment:
OnPageFrameDetached?

Reply:
this https://github.com/kblok/playwright/blob/v0.10.0/src/firefox/ffPage.ts#L57 & https://github.com/kblok/playwright/blob/v0.10.0/src/firefox/ffPage.ts#L127